components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateRangePicker.jsx

```diff
@@ -737,6 +737,33 @@ export default class DateRangePicker extends Widget {
         return result;
     }
 
+    getSupportedTimeRanges() {
+        const minGranularity = this.state.options.availableGranularities || 'From Second';
+        let timeRanges = [];
+        switch (minGranularity) {
+            case 'From Second':
```

Comment:
Do we need this case since this does nothing ?

Reply:
This is because the same output is given for both the cases 'From Second' and 'From Minute'